src/control/server/harness_test.go

```diff
@@ -97,8 +100,11 @@ func TestServer_Harness_Start(t *testing.T) {
 		},
 		"startup/shutdown with preset ranks": {
 			trc: &engine.TestRunnerConfig{
-				RunnerExitInfoCb: func() *engine.RunnerExitInfo {
-					time.Sleep(testLongTimeout)
+				RunnerExitInfoCb: func(ctx context.Context) *engine.RunnerExitInfo {
```

Comment:
this could probably be extracted as a function variable in the test to avoid the repetition